library/src/skins/default/src/sass/modules/tool/portal/_portal.scss

```
@@ -288,6 +291,13 @@ a[href]:not(.btn):hover {
 
   .site-link-block:hover {
     background-color: var(--site-nav-btn-hover-bg);
+    color: var(--site-nav-btn-current-hover-color);
+    a {
+      color: inherit;
+    }
+    i {
+      color: inherit;
+    }
```

We can make it more concise? 
```suggestion

  a, i {
    color: inherit;
  }
```